Algebraic Topology/algebraic-topology.tex

```diff
@@ -454,9 +454,194 @@ \section{Functoriality of the fundamental group}
+
+In general, I have many gluing maps $\phi_\alpha\from S^1 \to X_1$ and I glue $D^2_\alpha$ along $\phi_\alpha(S^1)$.  So $\fund(X_2) \isom \fund(X_1)/\langle[\phi_\alpha]\rangle$.  ((this one makes my head spin.  i dont get it))
+
+\begin{thm}[Not to be proven]
+  Every group is the fundamental group of some CW complex.  Construct as follows:
+
+  $X_0$ = point
+
+  $X_1$ = every element in the group gets a loop (the free group)
+
+  $X_2$ = every entry in the multiplication table gets a disk (a relation)
+
+  So it's just like a group presentation!
+\end{thm}
+\begin{thm}
+  Every finitely presented group is the fundamental group of a finite CW complex.
+\end{thm}
+\begin{thm}
+  If $n > 3$, then for any finitely presented group $G$, there exists
+  a compact manifold $M$ such that $G \isom \fund(M)$.
+\end{thm}
+\begin{proof}
+  If $n > 2$ and $M$ and $N$ are $n$-manifolds, then consider $M \#
+  N$. Recall that $\fund(M \# N) \isom \fund(M) * \fund(N)$. In
+  particular, we note that $\fund(\#^n (S^{n-1} \times S^1)) \isom
+  F_n$. 
+
+  % \includegraphics{} ((that puncture-glueing thing))
+
+  Finally, note that $\d(S^0 \x D^n) = S^0 \x S^{n-1} \setminus
+  \d(D^1 \x S^{n-1})$ (If you really want, you can break these
+  previous facts into lemmas.)
+
+  % Here is where the actual proof starts.
+  Let $M = \#^r (S^1 \times S^{n-1})$ be an $n$-manifold and $n > 3$
+  with $r$ the number of generators in the presentation of our group.
+  Now, consider a non-self-intersecting loop 
+
+  $\alpha \subset U \isom S^1 \x D^{n-1}$ where $U$ comes from the
+  tubular neighborhood theorem applied to $\alpha$. 
+
+  Then, we get that $\d(D^2 \x S^{n-2}) \isom \d U \isom S^1 \x
+  S^{n-2}$. 
+
+  % For $n = 2$
+
+  % <picture of cylindar> ((yeah so basically it's this proof that doesn't make sense to me at all.))
+
+  Thus, $\widetilde{M} = \frac{{(M \setminus U) \dunion (D^2 \x S^{n-2})}}{\sim} = \left( M
+    \setminus U \right) \union \left( D^2 \times S^{n-2}
+  \right)$. Now, to complete the proof, we wish to compute
+  $\fund(\tilde{M})$ using Van Kampen's theorem. So, this tells us
+  that \[
+    \fund(\widetilde{M}) \isom \frac{\fund(M \setminus U) * \fund(D^2
+      \times S^{n-2})}{\fund((M \setminus U) \cap (D^2 \times S^{n-2}))}.
+  \]
+  So, we note that $(M \setminus U) \cap (D^2 \times S^{n-2})$ is
+  homotopically equivalent to $S^1 \times S^{n-2}$ and that $(M
+  \setminus U) \# (D^2 \times S^{n-2})$ is homeomorphic to
+  $M$. Thus, \[
+\fund(\widetilde{M}) \isom \frac{\fund(M \setminus U) * \fund(D^2
+      \times S^{n-2})}{\fund((M \setminus U) \cap (D^2 \times
+      S^{n-2}))} \isom \frac{\fund((M \setminus U) \# (D^2 \times
+      S^{n-2}))}{\fund(S^1 \times S^{n-2})} \isom \frac{\fund(M)}{\langle [\alpha] \rangle}
+  \]
+  and thus, $\fund(\widetilde{M})$ is isomorphic to the free group on
+  our generators with one relation given by $[\alpha]$. We can repeate
+  this process repeatedly until we get all our relations.
```

Comment:
So, I think a good write up of this proof can be found here: https://chiasme.wordpress.com/2014/05/28/amalgamated-products-and-hnn-extensions-iv-markov-properties/

It might be worth reading that first (just skip to the appropriate theorem) to get an idea of what is actually going on here.